components/automate-ui/src/app/entities/service-groups/service-groups.requests.ts

```diff
@@ -1,15 +1,23 @@
 import { HttpClient, HttpParams } from '@angular/common/http';
 import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { reduce } from 'lodash/fp';
 import { Injectable } from '@angular/core';
 
 import {
-  ServiceGroupsPayload, ServiceGroupFilters,
+  ServiceGroupsPayload,
+  ServiceGroupFilters,
   HealthSummary,
-  ServicesPayload, ServicesFilters
+  ServicesPayload,
+  ServicesFilters
 } from './service-groups.model';
 import { environment } from '../../../environments/environment';
 const APPLICATIONS_URL = environment.applications_url;
 
+interface RespSuggestion {
```

Comment:
can we move this definition to the `service-groups.model.ts` file?

Reply:
@afiune updated 👍🏻